ls1intum/Artemis
# added pipe to dates in order to show them in a readable format

### Checklist

### Motivation and Context


[issue #1362](https://github.com/ls1intum/Artemis/issues/1362)

### Description
Added date pipe to display date in a readable format

### Steps for Testing

1. Go to 'Course Management'
2. Scroll on a course you created
3. Click on 'Lectures'
4. Click on 'View'
5. See error

### Screenshots
![Screenshot 2020-04-28 at 17 38 40](https://user-images.githubusercontent.com/33379863/80507427-226e7800-8977-11ea-80f2-540237fefc62.png)
